ledger/participant-integration-api/src/main/scala/platform/store/backend/h2/H2ResetStorageBackend.scala

```diff
@@ -24,26 +24,16 @@ object H2ResetStorageBackend extends ResetStorageBackend {
           |truncate table party_entries;
           |truncate table string_interning;
           |truncate table participant_events_create_filter;
+          |truncate table transaction_metering;
           |set referential_integrity true;""".stripMargin)
       .execute()(connection)
     ()
   }
 
   override def resetAll(connection: Connection): Unit = {
+    reset(connection)
```

Comment:
does this has to do something with the metering implementation? or just plain refactoring?

if this is just plain refactoring, how about:
making the execution mechanics reusable as well?
reuse on a string level?
propagating this type of reuse approach to the other DB implementations as well?

Reply:
This has been resolved, there is no longer a `reset` function.